packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -151,6 +151,78 @@ export interface CronPayload {
   cron_org: string;
 }
 
+export interface BotSecrets {
+  privateKey: string;
+  appId: string;
+  secret: string;
+}
+
+/**
+ * A helper for fetch secret from SecretManager.
+ */
+async function getBotSecrets(): Promise<BotSecrets> {
```

Comment:
Do we want this method and `BotSecrets` exported publicly? This file is the main file for this library so someone can do:
```ts
 import {getBotSecrets} from `gcf-utils`;
```

If it's not intended to be publicly exported (exported for testing), let's move into a separate file.

Reply:
Exporting sounds good